static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -150,12 +161,26 @@ const requestBids = (
             () =>
                 new Promise(resolve => {
                     window.pbjs.que.push(() => {
+                        // TODO: Replace with commercial core's API
+                        // recordFirstPrebidStarted();
+                        // EventTimer.trigger('prebidStart');
```

Comment:
@ashishpuliyel can you review this I get an error that trigger is not a function, is that the correct way of using the API? Commented out temporarily 
<img width="1009" alt="Screenshot 2021-02-02 at 17 45 41" src="https://user-images.githubusercontent.com/51630004/106625154-14412e00-657f-11eb-9c90-57eddf085e1f.png">